app/views/layouts/_header.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if current_provider %>
-  <%= render(partial: "shared/partials/cookie_banner") if current_provider.cookies_enabled.nil? %>
+  <%= render(partial: "shared/partials/cookie_banner") if display_cookie_banner(current_provider) %>
```

Comment:
I think something has changed and caused the errors. The call here does not match the method name in the helper
```suggestion
  <%= render(partial: "shared/partials/cookie_banner") if display_cookie_banner?(current_provider) %>
```